localization/mt.xml

```diff
@@ -104,92 +106,94 @@
-      <taxRule iso_code_country="es" id_tax="4"/>
-      <taxRule iso_code_country="fr" id_tax="4"/>
-      <taxRule iso_code_country="ie" id_tax="4"/>
-      <taxRule iso_code_country="it" id_tax="4"/>
-      <taxRule iso_code_country="cy" id_tax="4"/>
-      <taxRule iso_code_country="lv" id_tax="4"/>
-      <taxRule iso_code_country="lt" id_tax="4"/>
-      <taxRule iso_code_country="lu" id_tax="4"/>
-      <taxRule iso_code_country="hu" id_tax="4"/>
-      <taxRule iso_code_country="mt" id_tax="4"/>
-      <taxRule iso_code_country="nl" id_tax="4"/>
-      <taxRule iso_code_country="at" id_tax="4"/>
-      <taxRule iso_code_country="pl" id_tax="4"/>
-      <taxRule iso_code_country="pt" id_tax="4"/>
-      <taxRule iso_code_country="ro" id_tax="4"/>
-      <taxRule iso_code_country="si" id_tax="4"/>
-      <taxRule iso_code_country="sk" id_tax="4"/>
-      <taxRule iso_code_country="fi" id_tax="4"/>
-      <taxRule iso_code_country="se" id_tax="4"/>
-      <taxRule iso_code_country="uk" id_tax="4"/>
-    </taxRulesGroup>
-    <taxRulesGroup name="EU VAT For Virtual Products" auto-generated="1" eu-tax-group="virtual">
-      <taxRule iso_code_country="mt" id_tax="1"/>
-      <taxRule iso_code_country="at" id_tax="5"/>
-      <taxRule iso_code_country="be" id_tax="6"/>
-      <taxRule iso_code_country="bg" id_tax="7"/>
-      <taxRule iso_code_country="cy" id_tax="8"/>
-      <taxRule iso_code_country="cz" id_tax="9"/>
-      <taxRule iso_code_country="de" id_tax="10"/>
-      <taxRule iso_code_country="dk" id_tax="11"/>
-      <taxRule iso_code_country="ee" id_tax="12"/>
-      <taxRule iso_code_country="es" id_tax="13"/>
-      <taxRule iso_code_country="fi" id_tax="14"/>
-      <taxRule iso_code_country="fr" id_tax="15"/>
-      <taxRule iso_code_country="gb" id_tax="16"/>
-      <taxRule iso_code_country="gr" id_tax="17"/>
-      <taxRule iso_code_country="hr" id_tax="18"/>
-      <taxRule iso_code_country="hu" id_tax="19"/>
-      <taxRule iso_code_country="ie" id_tax="20"/>
-      <taxRule iso_code_country="it" id_tax="21"/>
-      <taxRule iso_code_country="lt" id_tax="22"/>
-      <taxRule iso_code_country="lu" id_tax="23"/>
-      <taxRule iso_code_country="lv" id_tax="24"/>
-      <taxRule iso_code_country="nl" id_tax="25"/>
-      <taxRule iso_code_country="pl" id_tax="26"/>
-      <taxRule iso_code_country="pt" id_tax="27"/>
-      <taxRule iso_code_country="ro" id_tax="28"/>
-      <taxRule iso_code_country="se" id_tax="29"/>
-      <taxRule iso_code_country="si" id_tax="30"/>
-      <taxRule iso_code_country="sk" id_tax="31"/>
-    </taxRulesGroup>
-  </taxes>
-  <units>
-    <unit type="weight" value="kg"/>
-    <unit type="volume" value="L"/>
-    <unit type="short_distance" value="cm"/>
-    <unit type="base_distance" value="m"/>
-    <unit type="long_distance" value="km"/>
-  </units>
-</localizationPack>
+      <taxRule iso_code_country="hr" id_tax="3">
+        <taxRule iso_code_country="es" id_tax="3"/>
```

Comment:
@ziegenberg It seems there are some additional spaces to remove.

Reply:
A "/" in line 109 was missing. I'll correct my PR.